### ISSUE TITLE
Grindr forces me to update the app?

### ISSUE THREAD

**Issue by matias200308** [ElJaviLuki/GrindrPlus#18]
Before knowing this mod, every time a new version of Grindr came out the app would stop working until I updated it, like a connection error, it happened to me with version 8.12 when version 8.12.1 came out, and it happened again when it came out version 8.13.  when I found this mod and had to go back to version 8.11 to use it I didn't have a problem again, but now that the mod was updated to version 8.17 the same problem is happening to me, I can't use Grindr without updating to version 8.17.1 , I get a connection error.  for now I went back to version 8.16 which works for me, but I would like to know if this problem occurs to someone else or is it a problem with my device.

sorry for my bad english, i only speak spanish, google translate is to blame 🤣

**Comment by ElJaviLuki** [comment]
So far, I didn't update my Grindr client and works perfectly fine

**Comment by Zarkoob** [comment]
Yeah if you go to 8.17.1 it won't work. Gotta follow this apps directions. It's not released for 8.17.1 (yet)

**Comment by Lanchon** [comment]
8.17.0 works fine for me.

maybe u enrolled in beta testing? maybe that forces the eager update on your device?

**Comment by zsantino** [comment]
Same situation for me. Forced to update to V8.17.1. I changed the version in the file app\src\main\java\com\eljaviluki\grindrplus\Hooker.kt, make an .apk with android studio and it's OK. But I'm not a programmer, I didn't check absolutely anything so I didn't want to publish my modification...

**Comment by ElJaviLuki** [comment]
> Same situation for me. Forced to update to V8.17.1. I changed the version in the file app\src\main\java\com\eljaviluki\grindrplus\Hooker.kt, make an .apk with android studio and it's OK. But I'm not a programmer, I didn't check absolutely anything so I didn't want to publish my modification...

I'm happy to tell you, you were -luckily- right!

**Comment by Lanchon** [comment]
im still using 8.17.0 with no issues
